src/energyplus/ForwardTranslator/ForwardTranslatePlantEquipmentOperationSchemes.cpp

```diff
@@ -158,6 +159,28 @@ namespace energyplus {
     return false;
   }
 
+  openstudio::energyplus::ComponentType convertOSEnumToEnum(openstudio::ComponentType compType) {
+    if (compType == openstudio::ComponentType::Cooling) {
+      return openstudio::energyplus::ComponentType::COOLING;
+    } else if (compType == openstudio::ComponentType::Heating) {
+      return openstudio::energyplus::ComponentType::HEATING;
+    } else if (compType == openstudio::ComponentType::Both) {
+      return openstudio::energyplus::ComponentType::BOTH;
+    }
+
+    return openstudio::energyplus::ComponentType::NONE;
+  }
+
+  openstudio::energyplus::ComponentType plantLoopType(const PlantLoop& plantLoop) {
+    LOG_FREE(Warn, "openstudio.energyplus.plantLoopType", "Method is deprecated. Use Loop::componentType instead");
+    return convertOSEnumToEnum(plantLoop.componentType());
+  }
+
+  openstudio::energyplus::ComponentType componentType(const openstudio::model::HVACComponent& component) {
+    LOG_FREE(Warn, "openstudio.energyplus.componentType", "Method is deprecated. Use HVACComponent::componentType instead");
+    return convertOSEnumToEnum(component.componentType());
+  }
```

Comment:
Deprecated functions in FT now call the model API. I have to do enum shenanigans since the FT had an `enum class` and now we use an OPENSTUDIO_ENUM (which is a class)